cmsplugin_cascade/templatetags/cascade_tags.py

```diff
@@ -70,12 +71,21 @@ def render_tag(self, context, plugin):
         if not plugin:
             return ''
 
-        content_renderer = context['cms_content_renderer']
-        content = content_renderer.render_plugin(
-            instance=plugin,
-            context=context,
-            editable=content_renderer.user_is_on_edit_mode(),
-        )
+        try:
```

Comment:
since this is a compatibility issue with django-CMS, wouldn't it be nicer to do a version check rather than an exception handling.
One day versions 3.4.x and lower will be gone and then nobody remembers why this exception handler was introduced. With a version check this would be obvious right from the beginning.